lib/shape/mxText.d.ts

```diff
@@ -40,7 +42,7 @@ declare module 'mxgraph' {
       color?: string,
       family?: string,
       size?: number,
-      fontStyle?: string | 0,
+      fontStyle?: number,
```

Comment:
This may introduce an inconsistency with other fontStyle function that takes a string parameter.
If this is an issue, these functions should updated as well to take an `number` parameter

Font style values are always numbers and are often manipulated in bitwise operations
https://github.com/jgraph/mxgraph/blob/v4.2.2/javascript/src/js/shape/mxText.js#L1275-L1283

https://github.com/jgraph/mxgraph/blob/81dcd5cc86d48792c194ebaa3437a7ebb7a2f9d1/javascript/src/js/shape/mxText.js#L1275-L1283